Formula/v/visp.rb

```diff
@@ -155,14 +155,25 @@ def post_install
     # Replace SDK paths in bottle when pouring on different OS version than bottle OS.
     # This avoids error like https://github.com/orgs/Homebrew/discussions/5853
     # TODO: Consider handling this in brew, e.g. as part of keg cleaner or bottle relocation
-    if OS.mac? && Tab.for_formula(self).poured_from_bottle && MacOS.version != bottle&.tag&.to_macos_version
+    if OS.mac? && (tab = Tab.for_formula(self)).poured_from_bottle
+      bottle_os = bottle&.tag&.to_macos_version
+      if bottle_os.nil? && (os_version = tab.built_on.fetch("os_version", "")[/\d+(?:\.\d+)*$/])
+        bottle_os = MacOSVersion.new(os_version).strip_patch
+      end
+      return if bottle_os.nil? || MacOS.version == bottle_os
+
       sdk_path_files = [
         lib/"cmake/visp/VISPConfig.cmake",
         lib/"cmake/visp/VISPModules.cmake",
         lib/"pkgconfig/visp.pc",
       ]
-      bottle_sdk_path = MacOS.sdk_for_formula(self, bottle&.tag&.to_macos_version).path
-      inreplace sdk_path_files, bottle_sdk_path, MacOS.sdk_for_formula(self).path, audit_result: false
+      old = MacOS.sdk_for_formula(self, bottle_os).path
+      new = MacOS.sdk_for_formula(self).path
+      sdk_path_files.each do |sdk_path_file|
+        # NOTE: Should be `odebug` to match os/mac/keg.rb but brew postinstall doesn't support it
+        ohai "Changing SDK path in #{sdk_path_file}", "  from #{old}\n    to #{new}"
+        inreplace sdk_path_file, old, new, audit_result: false
+      end
```

Comment:
Thinking over stdout, I will probably switch back to no output as that is what most users are used to:
```suggestion
      inreplace sdk_path_files, old, new, audit_result: false
```

Will try another run afterward.